components/StyledRadioList.js

```diff
@@ -44,7 +44,7 @@ const StyledRadioList = ({ children, id, name, onChange, options, keyGetter, dis
                 id={id && key + id}
                 value={key}
                 defaultChecked={props.defaultValue !== undefined && defaultValueStr === key}
-                disabled={disabled}
+                disabled={value.disabled || disabled} // disable a specific option or entire options
```

Comment:
I think that value may be `null` sometimes (it's valid to pass `options=[{key: 'oneTime', value: null}]`), let's be more defensive

```suggestion
                disabled={disabled || (value && value.disabled)} // disable a specific option or entire options
```